.github/workflows/merge_to_master.yml

```diff
@@ -1,10 +1,7 @@
 # CI stages to execute against master branch on PR merge
 name: update_robottelo_image
 
-on:
-  push:
-    branches:
-      - master
```

Comment:
I guess specifying branch for on push isn't required for master as well, as it should run for all branches.

Please correct me if not or let me know if the same changes should be backported for master?

Reply:
I think we should explicitly list branches that we want to push for - because the pyup.io bot creates branches on the main fork, so I think this would end up running against those branches.

Reply:
we need to have `- 6.9.z` instead of `- master`